### DISCUSSION TITLE
sh: /vercel/path0/node_modules/.bin/tailwindcss: Permission denied

### DISCUSSION THREAD

**Discussion by israpasos** [vercel/community#1237] [Help]
### Summary

I'm trying to deploy a Nextjs project, part of a turborepo. The nextjs project uses a package in the monorepo called ui which uses tailwind. Everything works on local but when I deploy to Vercel I get the following error:

`sh: /vercel/path0/node_modules/.bin/tailwindcss: Permission denied`

Answers on SO suggest chmod and pnpm rebuild. Tried adding different command as part of the install override with no luck.

Any ideas on what might be causing this?

**Comment by amyegan** [comment]
Thanks for reaching out @TzuTech! Is your repository public, or do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that you can share? I'm not sure what is causing this, so I need to get a better look at what is going wrong. 🙏
